src/ui/components/MapBox/MapBox.js

```diff
@@ -0,0 +1,163 @@
+ui.class.Mapbox = class Mapbox extends HTMLElement {
+	constructor(props) {
+		super();
+		const self = this;
+
+		let m_parentTag;
+		let m_containerTag;
+
+		let m_props;
+		let m_map;
+
+		this.addPoints = addPoints;
+		this.setup = setup;
+
+		if (props) {
+			checkMapboxLibraries();
```

Comment:
Move its contents inside setup{} function, execute them first and delete function checkMapboxLibraries{}